README.md

```diff
@@ -14,33 +14,35 @@
   - 4.1. [Heroku](#Heroku)
   - 4.2. [Abraxas](#Abraxas)
   - 4.3. [Docker (anywhere)](#Dockeranywhere)
-- 5. [E2E tests](#E2Etests)
-- 6. [GraphQL performance tests](#GraphQLperformancetests)
-  - 6.1. [Automation](#Automation)
-  - 6.2. [How to add or modify the tests](#Howtoaddormodifythetests)
-- 7. [Load tests](#Loadtests)
-  - 7.1. [Automation](#Automation-1)
-  - 7.2. [Local setup](#Localsetup)
-  - 7.3. [Running the tests locally](#Runningthetestslocally)
-  - 7.4. [Recording the tests using Playwright](#RecordingthetestsusingPlaywright)
-- 8. [Authentication](#Authentication)
-  - 8.1. [Locally](#Locally)
+- 5. [Developing GitHub Actions](#DevelopingGitHubActions)
+- 6. [E2E tests](#E2Etests)
+- 7. [GraphQL performance tests](#GraphQLperformancetests)
+  - 7.1. [Automation](#Automation)
+  - 7.2. [How to add or modify the tests](#Howtoaddormodifythetests)
+- 8. [Load tests](#Loadtests)
+  - 8.1. [Automation](#Automation-1)
+  - 8.2. [Local setup](#Localsetup)
+  - 8.3. [Running the tests locally](#Runningthetestslocally)
+  - 8.4. [Recording the tests using Playwright](#RecordingthetestsusingPlaywright)
+- 9. [Authentication](#Authentication)
+  - 9.1. [Locally](#Locally)
+- 10. [Translations](#Translations)
 
 <!-- vscode-markdown-toc-config
 	numbering=true
 	autoSave=true
 	/vscode-markdown-toc-config -->
 <!-- /vscode-markdown-toc -->
 
-## Documentation
+## 1. <a name='Documentation'></a>Documentation
 
 A public documentation is available at https://visualize.admin.ch/docs/.
 
-## Development Environment
+## 2. <a name='DevelopmentEnvironment'></a>Development Environment
 
 To start the development environment, you need a Docker runtime, e.g. [Docker Desktop](https://www.docker.com/products/docker-desktop) and [Nix](https://nixos.org).
 
-### Setting up the dev environment
+### 2.1. <a name='Settingupthedevenvironment'></a>Setting up the dev environment
```

Comment:
Looks like all the titles were concatenated together? Why did you change to the `a` tag? 😄

Reply:
It was automatically done by the Markdown TOC plugin I have on vscode, when I updated the table of contents. The link is empty, it is so that the table of contents works.